tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -19,6 +19,7 @@ use base "y2x11test";
 use strict;
 use warnings;
 use testapi;
+use utils qw(type_string_slow_extended);
```

Comment:
IMHO, This is going to break static checks in travis

Reply:
Yes, I wonder how the travis build passed here. This shouldn't have been merged.